src/Admin/BaseFieldDescription.php

```diff
@@ -256,6 +261,11 @@ public function hasAssociationAdmin()
         return null !== $this->associationAdmin;
     }
 
+    final public static function clearFieldGetters()
```

Comment:
The only purpose of this method is to use it in tests?

Reply:
Theoretically it can be useful when you alter your object's methods via runkit or something, but then you whole other set of problems to deal with.

Practically yes, this method was added to accomodate for mocks of the same class having or not having different methods of getting the same property.

Reply:
IMO we should make it private or remove it until we have some use case for it. wdyt @greg0ire ? IMO it doesn't feel right to have a method for testing purposes. And it will make things harder if we decide to move to use Property Access instead of this code.

Reply:
Do you mean to change the existing tests then to avoid reusing the same class name when testing for different properties/methods?

Reply:
Oddly enough, now the tests pass even without this method being present. Removed.